python/src/scipp/compat/mantid.py

```diff
@@ -675,8 +675,8 @@ def convert_EventWorkspace_to_data_array(ws,
     # events that are not within bucket bounds. Consider using `bin` instead
     # of `bins`?
     edges = coords_labs_data['coords'][dim]
-    coords_labs_data['coords'][dim] = sc.concatenate(edges[dim, 0],
-                                                     edges[dim, -1], dim)
+    coords_labs_data['coords'][dim] = sc.concatenate(edges[dim, 0:1],
```

Comment:
Why extract ranges now?
Also comment line above about "buckets" should be refactored

Reply:
See PR descirption. I also added a comment in an update I just pushed.